api/envoy/extensions/transport_sockets/tls/v3/tls.proto

```diff
@@ -123,6 +123,20 @@ message CommonTlsContext {
     }
   }
 
+  // Similar to CertificateProvider above, but allows the plugin instances to be configured on the
+  // client side instead of being sent from the control plane.
+  message CertificateProviderInstance {
+    // Plugin instance name. This name must be defined in the client's configuration (e.g., a
+    // bootstrap file) to correspond to a plugin instance (i.e., the same data in the typed_config
+    // field that would be sent in the CertificateProvider message if the config was sent by the
+    // control plane). If not present, defaults to "default".
+    string plugin_instance_name = 1;
+
+    // Opaque name used to specify certificate instances or types. For example, "ROOTCA" to specify
+    // a root-certificate (validation context) or "TLS" to specify a new tls-certificate.
```

Comment:
> to specify a new tls-certificate.

Why "new"? If the same `plugin_instance_name` and `certificate_name` combination is used for a different cluster (or such) will the provider provide the same certificate or has to mint a new one?

Reply:
Would be good to mention this is optional depending on the plugin_instance i.e. some plugins may require it and others may ignore it.